Documentation/docs/contributing/code_of_conduct.md

```diff
@@ -357,7 +357,7 @@ Harassment will not be tolerated in any of its forms either. Specifically:
 
 Although the above list cannot be exhaustive, we will not tolerate
 discrimination based on any of the protected characteristics in our
-[Diversity Statement](motivation.md#diversity-statement).
+[Diversity Statement](motivation.md).
```

Comment:
This was working correctly?
https://github.com/InsightSoftwareConsortium/ITK/pull/4268#pullrequestreview-1695399904